packages/bpk-component-section-header/src/BpkSectionHeader.module.scss

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+$bpk-spacing-v2: true;
+
+.bpk-section-header {
```

Comment:
a couple of things here:
- not sure if the default text colours matches the one in Figma, if yes then all good otherwise we should specify the colour
- there's also an onDark variant which is supported on desktop too!

Reply:
I recommend checking the new Figma dev playground feature if you haven't already so you can play around with props and see how they are supposed to look like! It's great for engineers 😄

Reply:
For the text colours issue: I checked them, all of them matche the one in Figma.